src/platforms/apple/common/configuration/out-of-memory.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Out Of Memory
+sidebar_order: 10
+description: "Learn how to turn off Out Of Memory"
+---
+
+**Attention: This is a preview API available since 7.0.0-alpha.3. This API is not in stable state yet. It may be renamed, changed or even removed in a future version. You can [give us feedback](https://github.com/getsentry/sentry-cocoa/discussions/1016) before it gets stable.**
```

Comment:
```suggestion
**Attention: This is a preview API available since 7.0.0-alpha.3. This API is not in stable state yet. It may be renamed, changed, or even removed in a future version. You can [give us feedback](https://github.com/getsentry/sentry-cocoa/discussions/1016) before it becomes stable.**
```